ld/armv7a7-imx6ull.ldt

```diff
@@ -49,6 +57,26 @@ REGION_ALIAS("HEAP", m_ocram_low);
 REGION_ALIAS("STACK", m_ocram_low);
 REGION_ALIAS("PLO_IMAGE", m_ocram_high);
 REGION_ALIAS("BSS", m_ocram_high);
+REGION_ALIAS("NAND", m_ddr);
+
+SECTIONS
+{
+	.nand ADDR_NAND (NOLOAD) :
```

Comment:
This section is more about DMA than NAND, we might want use this section for other things too in the future